File: extract_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('Jake_s_Resume__Anonymous_.pdf');

pdf(dataBuffer).then(function(data) {
    console.log(data.text);
}).catch(function(error) {
    console.error("Error parsing PDF:", error);
});

File: extract_pdf.py
import PyPDF2
with open('Jake_s_Resume__Anonymous_.pdf', 'rb') as file:
    reader = PyPDF2.PdfReader(file)
    text = ""
    for page in reader.pages:
        text += page.extract_text() + "\n"
with open('resume_text.txt', 'w', encoding='utf-8') as out_file:
    out_file.write(text)
